net-p2p/retroshare/retroshare-0.6.5.ebuild

```diff
@@ -50,7 +50,7 @@ RDEPEND="
 DEPEND="${RDEPEND}
 	gui? ( dev-qt/designer:5 )
 	jsonapi? (
-		app-doc/doxygen
+		|| ( <app-doc/doxygen-1.8.16 >=app-doc/doxygen-1.8.17 )
```

Comment:
please note that newer version should always come first here.

Reply:
thanks, I'll take it in acocunt for next PR ;)